src/js/modules/rpc/service.ts

```diff
@@ -16,11 +16,14 @@ import LogService from "../utilities/Logging";
 
 // read yaml config file
 const readConfigFile = (confPath: string): Promise<Record<string, any>> => {
-  try {
-    return fs.promises.readFile(confPath).then((file) => safeLoadAll(file)[0]);
-  } catch (e) {
-    return Promise.reject(new Error(`Error reading config file: ${e.message}`));
-  }
+  return fs.promises
```

Comment:
Just learned by @andresaristizabal that this change doesn't fix anything, the exception was always caught. However should we still keep the change? To me it reads better, in an async context we use .catch, otherwise prefer try/catch.

Reply:
Sure, it is clearest.